README.md

```diff
@@ -13,6 +13,9 @@ Module Input Variables
 - `vpc_id` - The VPC ID to place the cluster in
 
 #### Optional
+note about `user_data` and - `additional_user_data_script`: The `user_data` parameter overwrites the user_data template used by this module, this will break some of the module features. (`docker_storage_size`, `dockerhub_token`, and `dockerhub_email`) `additional_user_data_script` however will concatenate additional data to the end of the current user_data script. It is recomended that you use `additional_user_data_script`. These two parameters are mutually exclusive - you can not pass both into this module and expect it to work.
```

Comment:
now i remember us talking about this; is there a particular reason why we permit passing `user_data` at all?  as opposed to only permitting `additional_user_data_script`?

if someone REALLY REALLY hates what we do in our stock user data script, they can certainly undo it in an additional user data script, and then they've broken it and they get to keep the pieces.